tests/console/check_system_info.pm

```diff
@@ -104,7 +123,7 @@ sub run {
         # For hpc, system doesn't include legacy module
         $myaddons =~ s/lgm,?//g if (get_var("SCC_ADDONS", "") =~ /hpcm/);
         check_addons($myaddons);
-        check_product;
+        check_product("after");
```

Comment:
Any parameters defined as "after"?

Reply:
It need input some parameter for this function. 'before' means before migration. Any other word means after migration.  I want to make it clear, so use 'after' as input.

Reply:
Yutao, I am not a an expert on this, but I'd suggest you define it in function. depend on you